### PR TITLE
Sarinali/cfd 82 Update Comment Component

### PR DESCRIPTION
**Pull request by sarinali**
### Description
High Fi design for comment component

### Reason for Change
CFD-82

Completes CFD-82

### Type of change
- [x] New feature (non-breaking change which adds functionality)

IMG_5930
IMG_5929

### Testing
```const data = {
    id: 0,
    postId: 0,
    text: "Lorem Ipsum",
    createdAt: new Date(start),
    userId: 1,
  }```
for the comment component `<Comment comment={data}></Comment>` in `index.tsx`

### CODE REVIEW

File: apps/expo/src/components/Comment.tsx [hunk 1]
```diff
@@ -42,14 +42,18 @@ export default function Comment({ comment }: { comment: CommentComponent }) {
...
           }}
-          className="h-8 w-8 rounded-full"
+          className="h-12 w-12 rounded-full"
           resizeMode="cover"
         />
-        <Text className="ml-2 mr-2 text-base">{comment.userId}</Text>
-        <Text>Icon</Text>
-        <Text className="ml-2 text-base">{formatElapsed()}</Text>
+        <View>
+          <Text className="font-poppins400 ml-4 text-base">John Doe</Text>
```
Comment (BakerWJ) [inline apps/expo/src/components/Comment.tsx:49]: for text size, use the custom sizes that we created in tailwind: https://github.com/uoftblueprint/centre-for-dreams/pull/212

File: apps/expo/src/components/Comment.tsx [hunk 2]
```diff
@@ -42,14 +42,18 @@ export default function Comment({ comment }: { comment: CommentComponent }) {
...
+          <Text className="font-poppins400 ml-4 text-slate-500">
```
Comment (BakerWJ) [inline apps/expo/src/components/Comment.tsx:50]: for the color, use the color variants that we added to tailwind: https://github.com/uoftblueprint/centre-for-dreams/pull/219

**Review by BakerWJ — CHANGES_REQUESTED** [review]

File: apps/expo/src/components/Comment.tsx (hunk 1 above)
Comment (sarinali) [reply apps/expo/src/components/Comment.tsx]: ok

File: apps/expo/src/components/Comment.tsx (hunk 2 above)
Comment (sarinali) [reply apps/expo/src/components/Comment.tsx]: ok

File: apps/expo/src/components/Comment.tsx [hunk 3]
```diff
@@ -42,14 +42,16 @@ export default function Comment({ comment }: { comment: CommentComponent }) {
...
-        <Text>Icon</Text>
-        <Text className="ml-2 text-base">{formatElapsed()}</Text>
+        <View>
+          <Text className="font-title-md ml-4">John Doe</Text>
+          <Text className="font-body-md text-n-60 ml-4">{formatElapsed()}</Text>
+        </View>
       </View>
-      <Text className="ml-10 mt-2">{comment.text}</Text>
+
+      <Text className="ffont-body-md ml-16 mt-2">{comment.text}</Text>
```
Comment (BakerWJ) [inline apps/expo/src/components/Comment.tsx:54]:
```suggestion
      <Text className="font-body-md ml-16 mt-2">{comment.text}</Text>
```

**Review by BakerWJ — COMMENTED** [review]
looks good -- just need to fix typo

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**